src/devices/Bh1750fvi/Bh1750fvi.cs

```diff
@@ -0,0 +1,109 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Buffers.Binary;
+using System.Device.I2c;
+using System.Threading;
+
+namespace Iot.Device.Bh1750fvi
+{
+    /// <summary>
+    /// Ambient Light Sensor BH1750FVI
+    /// </summary>
+    public class Bh1750fvi : IDisposable
+    {
+        private const byte DefaultLightTransmittance = 0b_0100_0101;
+
+        private I2cDevice _sensor;
+
+        private double _lightTransmittance;
+        /// <summary>
+        /// BH1750FVI Light Transmittance, from 27.20% to 222.50%
+        /// </summary>
+        public double LightTransmittance
+        {
+            get => _lightTransmittance;
```

Comment:
should we read this from register?

Reply:
The sensor doesn't seem to provide a register address table, it just gives the setup command...